### ISSUE TITLE
NullPointerException bitBucketServer

### ISSUE THREAD

**Issue by vchernyshov** [danger/kotlin#213]
I'm getting this error:
```
java.lang.NullPointerException

        at systems.danger.kotlin.models.danger.DangerDSL.getBitBucketServer(DangerDSL.kt:27)
        at Dangerfile_df.<init>(Dangerfile.df.kts:9)
```
This is my danger file:
```
@file:Repository("https://repo.maven.apache.org")
@file:DependsOn("org.apache.commons:commons-text:1.6")

import org.apache.commons.text.WordUtils
import systems.danger.kotlin.*
import systems.danger.kotlin.models.danger.DangerDSL

danger(args) {
    if (bitBucketServer.pullRequest.title.contains("WIP", false)) {
        warn("PR is classed as Work in Progress")
    }
}
```

Any ideas why it could happen?

**Comment by f-meloni** [comment]
It seems that the `bitBucketServer` object in your DSL is null.

What is the output of `danger-kotlin pr $YOUR_PR_LINK --json`?

**Comment by vchernyshov** [comment]
@f-meloni here is the json structure:
<img width="322" alt="Screenshot 2022-04-10 at 09 04 14" src="https://user-images.githubusercontent.com/5671443/162604678-1803a5bd-215d-42ae-84fb-543fd5c879dc.png">

according to source code ```_bitBucketServer ``` is serialised with key ```bitbucket_server``` so looks like ```danger-kotlin``` doesn't support  ```bitbucket_server```

**Comment by vchernyshov** [comment]
@f-meloni it's simple but time consuming, `bitbucket_cloud` has different API response so it's not possible just to add new property to Danger with `bitbucket_cloud`, I had to modify several DTOs to make it work. Currently it's enough for me to test Danger-Kotlin, if it will cover all our needs we age going to add support of `bitbucket_cloud` by ourself and create PR to Danger-Kotlin

**Comment by f-meloni** [comment]
Yeah, Danger Swift has already that DSL built, so we might just try to copy it here https://github.com/danger/swift/blob/master/Sources/Danger/BitBucketCloud.swift

**Comment by vchernyshov** [comment]
Thanks, it will help me.

**Comment by vchernyshov** [comment]
@f-meloni I have checked cloud support for swift and looks like the models are outdated.
I have extra question: is it possible to use pull request model inside plugin?
